o-clock.cabal

```diff
@@ -84,7 +84,9 @@ benchmark o-clock-benchmark
   ghc-options:         -Wall -Werror -O2 -threaded -rtsopts -with-rtsopts=-N
```

Comment:
Could you try to add `-O2` option to library temporary and see whether it changes something or not?

Reply:
It doesn't influence on the results, so I'm not adding it :slightly_smiling_face: